goosebit/api/v1/devices/routes.py

```diff
@@ -48,4 +48,32 @@ async def devices_delete(_: Request, config: DevicesDeleteRequest) -> StatusResp
     return StatusResponse(success=True)
 
 
+@router.patch(
+    "",
+    dependencies=[Security(validate_user_permissions, scopes=["device.write"])],
+)
+async def devices_patch(_: Request, config: DevicesPatchRequest) -> StatusResponse:
```

Comment:
Duplicated put endpoint. PATCH IMHO should fail if resource does not exist - which is not the case.

Reply:
Good call.  Not sure if that's in scope for this PR, can fix it here if you want.

Reply:
Fixed, patch now raises if the device does not exist.